acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb

```diff
@@ -10,4 +10,6 @@
   puppet_version = default_puppet_version
 end
 
+# Always force 3.x agents to use FOSS puppet install type
+nonmaster_agents.each { |agent| agent[:type] == 'foss' }
```

Comment:
Yeeep. Initially I fixed this issue via pry, ran the tests, and when that worked I did the initial faulty PR and ran the tests again. When they failed I flagged this with the DO NOT MERGE commit tag while I verified. Running the tests now with the fixes; assuming that works I'm going to force push on top of this whoopsie.